Patches/Dragon's Descent/ThingDefs_Races/Patch_Races_Animal_Common_Dragon.xml

```diff
@@ -0,0 +1,1121 @@
+				<xpath>Defs/ThingDef[defName="Blue_Dragon"]/race/wildness</xpath>
+				<value>
+					<wildness>0.89</wildness>
+				</value>
+			</li>
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/PawnKindDef[defName="Blue_Dragon"]/combatPower</xpath>
+				<value>
+					<combatPower>2190</combatPower>
+				</value>
+			</li>
+
+			<!-- Green_Dragon -->
+			<!-- Verbs -->
+			<!--<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Green_Dragon"]/verbs</xpath>
+				<value>
+					<verbs>
+						<li Class="CombatExtended.VerbPropertiesCE">
+							<verbClass>CombatExtended.Verb_ShootCE</verbClass>
+							<recoilAmount>0.5</recoilAmount>
+							<commonality>0.9</commonality>
+							<ai_IsBuildingDestroyer>true</ai_IsBuildingDestroyer>
+							<targetParams>
+								<canTargetLocations>true</canTargetLocations>
+							</targetParams>
+							<hasStandardCommand>true</hasStandardCommand>
+							<defaultProjectile>Projectile_DragonBreath</defaultProjectile>
+							<warmupTime>5</warmupTime>
+							<range>25</range>
+							<burstShotCount>25</burstShotCount>
+							<ticksBetweenBurstShots>2</ticksBetweenBurstShots>
+							<minRange>2</minRange>
+							<soundCast>DragonBreathShot</soundCast>
+							<muzzleFlashScale>1</muzzleFlashScale>
+						</li>
+					</verbs>
+				</value>
+			</li>-->
+			<!-- Tools -->
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Green_Dragon"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<cooldownTime>3.5</cooldownTime>
+							<power>86</power>
+							<capacities>
+								<li>Scratch</li>
+							</capacities>
+							<linkedBodyPartsGroup>FrontLeftPaw</linkedBodyPartsGroup>
+							<label>left claw</label>
+							<armorPenetrationSharp>24.09</armorPenetrationSharp>
+							<armorPenetrationBlunt>120.417</armorPenetrationBlunt>
+							<chanceFactor>0.25</chanceFactor>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<cooldownTime>3.5</cooldownTime>
+							<power>86</power>
+							<capacities>
+								<li>Scratch</li>
+							</capacities>
+							<linkedBodyPartsGroup>FrontRightPaw</linkedBodyPartsGroup>
+							<label>right claw</label>
+							<armorPenetrationSharp>24.09</armorPenetrationSharp>
+							<armorPenetrationBlunt>120.417</armorPenetrationBlunt>
+							<chanceFactor>0.25</chanceFactor>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<cooldownTime>4.41</cooldownTime>
+							<power>92</power>
+							<capacities>
+								<li>Bite</li>
+							</capacities>
+							<linkedBodyPartsGroup>Teeth</linkedBodyPartsGroup>
+							<label>teeth</label>
+							<armorPenetrationSharp>4.75</armorPenetrationSharp>
+							<armorPenetrationBlunt>301.042</armorPenetrationBlunt>
+							<chanceFactor>0.2</chanceFactor>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<cooldownTime>2.12</cooldownTime>
+							<power>111</power>
+							<capacities>
+								<li>Toxic_Bite</li>
```

Comment:
As above, would replace this with vanilla `ToxicBite` unless this mod's version does something really special